pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/controller/ControllerDescribeStreamCommand.java

```diff
@@ -122,18 +131,42 @@ public static CommandDescriptor descriptor() {
     }
 
     private SegmentHelper instantiateSegmentHelper(CuratorFramework zkClient) {
+
+        // hostMonitorEnable should be true for pks clusters
```

Comment:
Please, do not make system-specific comments. Also, if `hostMonitorEnabled(false)`, is this going to be usable with both standalone and real clusters?

Reply:
If `hostMonitorEnabled(false)` it may not be usable with real clusters. Keeping it `false` allows for us to be able to run it against standalone and in turn write integration tests for the same. Changing it back to `true`, will use this during testcases.